src/radar-chart/index.js

```diff
@@ -261,7 +353,10 @@ RadarChart.propTypes = {
     polygons: PropTypes.object
   }),
   tickFormat: PropTypes.func,
-  width: PropTypes.number.isRequired
+  width: PropTypes.number.isRequired,
+  renderAxesOverPolygons: PropTypes.bool,
+  onValueMouseOver: PropTypes.func,
+  onValueMouseOut: PropTypes.func
```

Comment:
Wow the thing I thought you were originally suggesting was an onSeriesMouseOver, this is great. While you are in here, will you add a onSeriesMouseOver? I think it should be pretty easy?

Reply:
`onSeriesMouseOver` is added, with a showcase and tests.